applications/plexos-hpc-walkthrough/Login-HPC.md

```diff
@@ -1,23 +1,31 @@
-## Login into NREL HPC, Peregrine
+## Login into NREL HPC, Eagle
 
 A supported set of instructions for logging into the NREL HPC is provided [here](https://hpc.nrel.gov/users/connect/ssh)
 
 Initiate at git-bash terminal session (on Windows) or a terminal (on Mac).
 
-If you are on a standard network with internet connection, like the NREL guest wireless, you will need to login to the gateway system.
-```
+If you are on a standard network with internet connection, e.g. the NREL guest wireless, or are using a non-NREL computer you will need to login to the gateway system.
```

Comment:
I suggest adding a header for "External Access" to make it a bit clearer. It may be better to switch the order too and have internal first?